src/_pytest/python.py

```diff
@@ -1508,18 +1508,17 @@ def _showfixtures_main(config: Config, session: Session) -> None:
                 tw.line()
                 tw.sep("-", f"fixtures defined from {module}")
                 currentmodule = module
-        if verbose <= 0 and argname[0] == "_":
+        if verbose <= 0 and argname.startswith("_"):
             continue
-        tw.write(argname, green=True)
+        tw.write(f"{argname}", green=True)
```

Comment:
Is this intended?

Reply:
Yes this inclusion is intended.

Reply:
Alright - I don't really see the point of using an f-string here (with only one single string variable in it), but this is too minor to block this PR.